frontend/src/routes/version.tsx

```diff
@@ -119,7 +119,7 @@ function DataSources() {
                 Data Sources
             </Typography>
             <Box sx={{ display: "flex", flexWrap: "wrap", gap: 2 }}>
-                {config.data_sources.map((source) => (
+                {config.beets_meta_sources.map((source) => (
```

Comment:
Im not too sure how I feel about this rename. Shouldn't it be clear we talk about beets? `data_sources` was descriptive enough for me and it mirrors how beets names these things.